sdcm/utils.py

```diff
@@ -196,6 +212,45 @@ def upload_file(cls, file_path):
             logger.debug("Unable to upload to S3: %s" % e)
             return ""
 
+    @classmethod
+    def download_files_by_testid_from_bucket(cls, bucket_name, testid):
+        filenames = cls.find_filename_by_testid(bucket_name, testid)
```

Comment:
didn't find definition of `find_filename_by_testid`

Reply:
mistype. fixed